tests/unit/forms/rsvp_json_test.py

```diff
@@ -0,0 +1,88 @@
+import json
+
+from funnel import forms
+
+valid_json = json.dumps(
```

Comment:
If you do `json.dumps` on a string, the output will be a string. You need `json.loads`.